pkgs/development/libraries/gdal/default.nix

```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
     wrapPythonPrograms
   '';
 
-  enableParallelBuilding = true;
+  enableParallelBuilding = false;
```

Comment:
Why?

Reply:
There were files missing in the installation before I set it to `false`.

Reply:
This is worth documenting in a comment and/or the commit message.